src/PKSim.Presentation/UICommands/ExportODEForRUICommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OSPSuite.Core.Extensions;
+using OSPSuite.Core.Services;
+using OSPSuite.Presentation.UICommands;
+using PKSim.Core.Model;
+using PKSim.Core.Services;
+
+namespace PKSim.Presentation.UICommands
+{
+   public class ExportODEForRUICommand : ActiveObjectUICommand<Simulation>
+   {
+      private readonly ISimulationExportTask _simulationExportTask;
+
+      public ExportODEForRUICommand(ISimulationExportTask simulationExportTask, IActiveSubjectRetriever activeSubjectRetriever) : base(activeSubjectRetriever)
+      {
+         _simulationExportTask = simulationExportTask;
+      }
+
+      protected override async void PerformExecute()
+      {
+         await _simulationExportTask.SecureAwait(x => x.ExportODEForR(Subject));
```

Comment:
I would throw "not implemented" exception right here. At the moment user is asked for folder first, and after the folder selection the exception appears.

Reply:
That's ok like that.
 That's just for dev anyways.

Reply:
ok.
btw: just seen that MAtlab-ODE Export in Pk-Sim is also defined for devs only. Why?

Reply:
Beacuse Matlab is dead (and also noone ever missed in in PKSim). Good like that IMO until proven otherwise